### PR TITLE
multiload: Do not add the same global variables within all graphs

### PR DESCRIPTION
**Pull request by rbuj**

### CODE REVIEW

File: multiload/load-graph.c [hunk 1]
```diff
@@ -106,27 +109,27 @@ load_graph_draw (LoadGraph *g)
...
+        multiload->net_threshold = multiload->net_threshold1;
         level = 1;
-        if (maxnet < g->net_threshold1)
+        if (maxnet < multiload->net_threshold1)
           level = 0;
       }
 
     //printf("level %d maxnet = %d ", level, maxnet);
-    maxnet = maxnet/g->net_threshold;
+    maxnet = maxnet/multiload->net_threshold;
```
Comment (raveit65) [inline multiload/load-graph.c:129]: Beautiful warning :)
https://travis-ci.org/github/mate-desktop/mate-applets/jobs/708166154#L4353
```
load-graph.c:129:20: warning: Division by zero

    maxnet = maxnet/multiload->net_threshold;
```

**Comment by raveit65** [comment]
PR runs fine, so only the warning should be fixed from my side.

File: multiload/load-graph.c (hunk 1 above)
Comment (rbuj) [reply multiload/load-graph.c:129]: The cppcheck report may be wrong, so I don't think it needs to check whether` multiload->net_threshold > 1`
```
maxnet = maxnet / MAX (g->net_threshold, 1);
```

**Comment by rbuj** [comment]
@raveit65 Is there anything pending?

**Comment by raveit65** [comment]
I will test it again  over the weekend.

File: multiload/global.h [hunk 2]
```diff
@@ -81,6 +75,17 @@ struct _MultiloadApplet
...
+    long cpu_time [NCPUSTATES];
+    long cpu_last [NCPUSTATES];
+    int cpu_initialized;
+
+    double loadavg1;
+    NetSpeed *netspeed_in;
+    NetSpeed *netspeed_out;
+    guint net_threshold1;
+    guint net_threshold2;
+    guint net_threshold3;
```
Comment (sc0w) [inline multiload/global.h:88]: the indentation is wrong

Comment (rbuj) [reply multiload/global.h:88]: No it isn't

Comment (rbuj) [reply multiload/global.h:88]: take a look at _LoadGraph

Comment (rbuj) [reply multiload/global.h:88]: It looks like I had a deja vu, didn't we talk about it recently?

Comment (sc0w) [reply multiload/global.h:88]: seems do you like to see a weird file with misaligned indentations, I don't agree

Comment (rbuj) [reply multiload/global.h:88]: I formatted the rest of the file in a separate commit, I think there is no other PR that modifies the formatted code ;)

Comment (sc0w) [reply multiload/global.h:88]: Thanks!

**Pull request merged**